### ISSUE TITLE
Allow JSX fragments inside elements

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#679]
The following code is ~~illegal~~ legal, but quick-lint-js reports an error for it:
```js
return <div>
  text
  <>text</>
  text
</div>;
```

**Comment by strager** [comment]
Fixed in commit d80a061f76b901fb9700acf767327fdc09338f25. I shipped version 2.4.0 with the fix.
